src/app/dashboard/dashboard.component.ts

```diff
@@ -17,28 +9,6 @@ export class DashboardComponent implements OnInit {
     constructor() { }
 
     ngOnInit() {
-        const chartData = [{
-            mapping: 'JAX',
-            visits: 3
```

Comment:
Is this unused code?

Reply:
Yes, that array was not being used at all.